.github/workflows/terraform-daily-healthcheck-deploy-and-destroy.yml

```diff
@@ -1,90 +1,186 @@
-name: 'Daily Healthcheck Deploy and Destroy'
+name: 'Daily Healthcheck Deploy, Test and Destroy'
 
-#on:
-#  schedule:
-#    - cron: 45 7 * * 1-5
 on:
-  workflow_dispatch:
-    inputs:
-      buildBranch:
-        description: 'Feature branch to push to sandbox.'
-        required: true
-        type: 'string'
-      sandboxWorkspace:
-        description: 'Which Sandbox to push to.'
-        required: true
-        type: 'string'
-      environment:
-        default: 'development'
-        description: 'Which environment should this run against'
-        required: true
-        type: 'string'
-#
-#permissions:
-#  pull-requests: write
-#  id-token: write # This is required for requesting the JWT
-#  contents: read # This is required for actions/checkout
+ schedule:
+   - cron: 30 4 * * 1-5
+
+permissions:
+  pull-requests: write
```

Comment:
Does it need this permission?

Reply:
I am not sure I would assume no potentially I will try to remove it and check it runs ok